src/roiextractors/extractors/memmapextractors/memmapextractors.py

```diff
@@ -27,17 +27,19 @@ def __init__(
         self._video = video
         super().__init__()
 
-    def get_frames(self, frame_idxs=None):
+    def get_frames(self, frame_idxs=None, channel: int = 0):
```

Comment:
If `channel` can be `None` as an allowed input, add this to the annotation type as `Optional[int]`

Reply:
Good idea!